contents/docs/libraries/js/features.mdx

```diff
@@ -407,20 +407,20 @@ posthog.init('<ph_project_api_key>', {
 
         function redactObject(objectToRedact: Record<string, any>): Record<string, any> {
             return Object.entries(objectToRedact).reduce((acc, [key, value]) => {
-                const redactedValue = key.includes("url") && typeof value === "string" ? redactUrl(value) : value;
-                acc[redactUrl(key)] = redactedValue;
-                return acc;
+                if (key !== '_url' && (key.includes('url') || key === 'u') && typeof value === 'string') {
```

Comment:
`u` seems very vague... is it one of ours?

Reply:
like, it's only an example so 🤷

Reply:
It's part of `INITIAL_PERSON_INFO` which we accidentally send. I'll just remove that instead though